components/lvgl_ili9341/ili9341.c

```diff
@@ -166,27 +166,21 @@ void ili9341_enable_backlight(bool backlight)
 
 static void ili9341_send_cmd(uint8_t cmd)
 {
-	  while(disp_spi_is_busy()) {
-			taskYIELD();
-		};
+	  while(disp_spi_is_busy()) {};
```

Comment:
Semicolons are not necessary after braces.

Reply:
Fixed